Engine/src/Engine/ResourceManagement/Text.cpp

```diff
@@ -88,7 +88,7 @@ namespace Engine
 
 		m_whiteSpaceSize = Characters['H'].Size.x;
 
-		m_textShaderProgram = ShaderNorms::getInstance()->getShaderReference(ShaderFillType::FillType::SN_TEXT_FILL);
+		m_textShaderProgram = ShaderNorms::getInstance()->getShaderReference(ShaderFillType::FillType::FT_TEXTURE_FILL);
```

Comment:
For this, I needed to use a specific shader file for text rendering. Is there any way I can still reference that shader? It's called Text.vs/fs. It can be in a different PR, but I just want to bring it up now.

Reply:
So for this, you can add FT_TEXT_FILL to ShaderFillType module. Also check ShaderNorms() and ShaderNorms::getShaderNameString to add the text shader in accordingly.